images/builder/debug-wrapper.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+)
+
+func main() {
+	bin, err := os.Executable()
+	if err != nil {
+		log.Println(err)
+		os.Exit(1)
+	}
+
+	inputArts := os.Args[1:]
```

Comment:
nit, seems like a typo (though consistent elsewhere :sweat_smile: )
```suggestion
	inputArgs := os.Args[1:]
```

Reply:
thanks, fixed